VMEncryption/main/DiskUtil.py

```diff
@@ -192,73 +193,146 @@ def secure_key_release_operation(self,protectorbase64,kekUrl,operation,attestati
             return None
         self.logger.log("secure_key_release_operation {0} end.".format(operation))
         return process_comm.stdout.strip()
-    
-    def import_token(self,device_path,passphrase_file,public_settings):
-        '''this function reads passphrase from passphrase file, wrap it and update in token field of LUKS2 header.'''
+
```

Comment:
Please move this new functions to a new file for CVM.

Reply:
I have done it.